hooks/open-telemetry/pkg/metrics.go

```diff
@@ -25,18 +25,10 @@ type MetricsHook struct {
 	errorCounter   api.Int64Counter
 
 	flagEvalMetadataDimensions []DimensionDescription
+	attributeSetterCallback    *func(openfeature.FlagMetadata) []attribute.KeyValue
```

Comment:
nit: `attributeMapperCallback` or `attributeMapper` might be better, but that's totally up to you.

Reply:
Changed to `attributeMapperCallback`